app/main.py

```diff
@@ -1,3 +1,49 @@
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: int | float) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other: int | float):
+        if isinstance(other, Distance):
+            return Distance(self.km + other.km)
+        else:
+            return Distance(self.km + other)
+
+    def __iadd__(self, other: int | float):
+        if isinstance(other, Distance):
+            self.km += other.km
+            return self
+        else:
+            self.km += other
+            return self
+
+    def __mul__(self, other: float | int):
+        return Distance(self.km * other)
+
+    def __truediv__(self, other: float | int):
+        return Distance(round((self.km / other), 2))
+
+    def __lt__(self, other: int | float) -> bool:
+        if isinstance(other, (Distance, int, float)):
+            return self.km < other
```

Comment:
No check for isinstance. Fix in all places
```suggestion
        return self.km < other
```